### PR TITLE
Initial commit of the second newsletter. Still some TODOs for @japaric

### PR DESCRIPTION
**Pull request by jamesmunns**

### CODE REVIEW

**Comment by jamesmunns** [comment]
Closes https://github.com/rust-lang-nursery/embedded-wg/issues/65

**Comment by jamesmunns** [comment]
Rendered: https://github.com/rust-lang-nursery/embedded-wg/blob/newsletter-2/newsletters/2018-03-29.md

**Comment by japaric** [comment]
@jamesmunns I've thrown my summary into the PR. It still needs some editing (fix the headers, etc.). Let me know if you have comments or if anything is missing. I'll check this PR again in a few hours.

**Comment by japaric** [comment]
Another thing that I wanted to mention is that the linker issue we asked help for in the last newsletter already has a PR that will fix it in rust-lang/rust#49316.

File: newsletters/2018-03-29.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+This experimental feature has been in the compiler for a while and it's required in some scenarios when using LTO to prevent the compiler some function / `static` that needs to be in the final binary.
+
+We'll try to get it stabilized by the edition release but it's not a high priority feature.
+
+## Stability of the Embedded Targets
+
+We don't only want to make embedded Rust possible on stable; we also want to make sure the embedded targets don't regress. So we are going to add tests to rust-lang/rust CI to make sure regression block PRs from landing.
+
+That effectively will make some of the embedded targets into the tier 1 platform. The core team is fine with adding the `thumb*` targets (ARM Cortex-M) to tier 1. Less maintained, still in development and not fully mature targets like AVR, MSP430 and RISCV will become tier 3 -- they'll be tested but won't block PRs and rust-std binaries will be produced but it's not guaranteed there will be binaries available for all nightly / beta / stable releases.
```
Comment (jamesmunns) [inline newsletters/2018-03-29.md]: @japaric should AVR/MSP/RISCV be `tier 2` and not `tier 3`?

Comment (japaric) [reply newsletters/2018-03-29.md]: Now that I'm actually reading the description in the Rust Platform Support page yes; it sounds like those should be called tier 2 targets.

**Comment by japaric** [comment]
Thanks @jamesmunns!

**Pull request merged**